doc/getting-started.md

```diff
@@ -0,0 +1,25 @@
+# Getting Started
+
+> I'm just getting started with **m-ld**. I want to try building an app, without having to deploy a messaging service.
+
+We run a free public Socket.io messaging service for **m-ld**, on our cloud Gateway! Your domains will be named like `≪uuid≫.public.gw.m-ld.org`, and you'll need an internet connection to create new domains and keep clones synchronised (although individual clones can work offline, as usual). For usage instructions see [UUID subdomains](uuid-subdomains.md).
+
+---
+
+> I'm building a client-only browser or desktop app with **m-ld**. I want to offer secure backup of my users' data.
+
+On our cloud Gateway you can sign up for your own free account, and create domains which are backed-up there. Your domains will be named like `≪name≫.≪account≫.gw.m-ld.org`, and you'll need an internet connection to create new domains and keep clones synchronised (although individual clones can work offline, as usual). For usage instructions see [named subdomains](named-subdomains.md).
+
+---
+
+> I'm building an app with a service tier, using **m-ld** for data distribution.
+
+You can use our cloud Gateway to provide messaging and secure backup of your domains. You service tier can just talk to the cloud Gateway as if it were a client. To set your own service levels, or to work with a restricted network, you might choose to [self-host your own Gateway](self-host.md).
+
+---
+
+> I'm upgrading a legacy app, with a service tier and database, to offer live document sharing using **m-ld**.
```

Comment:
**suggestion:** This answer is a bit vague, and probably deserves some more detail, but not necessarily right now.

Reply:
Agreed. I will mark it as under construction.